Code/Tools/ProjectManager/Source/GemCatalog/GemItemDelegate.cpp

```diff
@@ -374,6 +374,45 @@ namespace O3DE::ProjectManager
         }
     }
 
+
+    void GemItemDelegate::DrawPlatformText(QPainter* painter, const QRect& contentRect, const QFont& standardFont, [[maybe_unused]] const QModelIndex& modelIndex) const
+    {
+        const GemInfo::Platforms platforms = GemModel::GetPlatforms(modelIndex);
```

Comment:
can we just early out here to avoid the rest of the logic?
```suggestion
        const GemInfo::Platforms platforms = GemModel::GetPlatforms(modelIndex);
        if(!platforms)
        {
            return;
        }
```